tests/test_transforms/test_math/test_binary_operator_transform.py

```diff
@@ -0,0 +1,320 @@
+        ("*", "feature", "target", "feature"),
+        ("/", "feature", "target", "feature"),
+        ("+", "target", "feature", "feature"),
+        ("-", "target", "feature", "feature"),
+        ("*", "target", "feature", "feature"),
+        ("/", "target", "feature", "feature"),
+    ],
+)
+def test_inverse_one_segment(ts_one_segment, operand, left_column, right_column, out_column):
+    target_vals = deepcopy(ts_one_segment.df["segment_0"][out_column].values)
+    transformer = binary_operator.BinaryOperationTransform(
+        left_column=left_column, right_column=right_column, operator=operand, out_column=out_column
+    )
+    new_ts = transformer.fit_transform(ts=ts_one_segment)
+    new_ts = transformer.inverse_transform(ts=new_ts)
+    new_ts_vals = new_ts.df["segment_0"][out_column].to_numpy()
+    numpy.testing.assert_array_almost_equal(new_ts_vals, target_vals)
+
+
+@pytest.mark.parametrize(
+    "operand, left_column, right_column, out_column",
+    [
+        ("+", "feature", "target", "target"),
+        ("-", "feature", "target", "target"),
+        ("*", "feature", "target", "target"),
+        ("/", "feature", "target", "target"),
+        ("+", "target", "feature", "target"),
+        ("-", "target", "feature", "target"),
+        ("*", "target", "feature", "target"),
+        ("/", "target", "feature", "target"),
+        ("+", "feature", "target", "feature"),
+        ("-", "feature", "target", "feature"),
+        ("*", "feature", "target", "feature"),
+        ("/", "feature", "target", "feature"),
+        ("+", "target", "feature", "feature"),
+        ("-", "target", "feature", "feature"),
+        ("*", "target", "feature", "feature"),
+        ("/", "target", "feature", "feature"),
+    ],
+)
+def test_inverse_two_segments(ts_two_segments, operand, left_column, right_column, out_column):
+    target_vals1 = deepcopy(ts_two_segments.df["segment_0"][out_column].values)
+    target_vals2 = deepcopy(ts_two_segments.df["segment_1"][out_column].values)
+    transformer = binary_operator.BinaryOperationTransform(
+        left_column=left_column, right_column=right_column, operator=operand, out_column=out_column
+    )
+    new_ts = transformer.fit_transform(ts=ts_two_segments)
+    new_ts = transformer.inverse_transform(ts=new_ts)
+    new_ts_vals1 = new_ts.df["segment_0"][out_column].to_numpy()
+    new_ts_vals2 = new_ts.df["segment_1"][out_column].to_numpy()
+    numpy.testing.assert_array_almost_equal(new_ts_vals1, target_vals1)
+    numpy.testing.assert_array_almost_equal(new_ts_vals2, target_vals2)
+
+
+@pytest.mark.parametrize(
+    "operand",
+    [
+        "/=",
+        "%",
+        "**",
+        "==",
+        ">=",
+        "<=",
+        ">",
+        "<",
+    ],
+)
+def test_inverse_failed(ts_one_segment, operand):
+    transformer = binary_operator.BinaryOperationTransform(
+        left_column="feature", right_column="target", operator=operand, out_column="target"
+    )
+    with pytest.raises(
```

Comment:
We should add error message (like here https://github.com/etna-team/etna/blob/4acf084c38e780951a78d20654eeacee984c733f/tests/test_transforms/test_decomposition/test_deseasonal_transform.py#L176)

May be we should rename this test to `test_inverse_faild_unsupported_operator` and add one more test for "inplace error"